iterators.tex

```diff
@@ -276,7 +276,7 @@
   template <class I>
   concept bool Readable@\newtxt{() \{}\oldtxt{ =}@
     @\newtxt{return}@ Semiregular<I>@\newtxt{()}@ &&
-      requires (I i) {
+      requires (@\newtxt{const }@I i) {
```

Comment:
I know it's not a problem here. It just raised a red flag. For good reason, considering `MoveWritable`.